perseo-main/src/main/java/com/telefonica/iot/perseo/Configuration.java

```diff
@@ -42,16 +42,21 @@ private Configuration() {
 
     private static final String DEFAULT_PERSEO_FE_URL = "http://127.0.0.1:9090";
     private static final long DEFAULT_MAX_AGE = 60000;
+    private static final long DEFAULT_INTERNAL_TIMER_MSEC_RESOLUTION = 10;
```

Comment:
New default value should be changed also in CNR and documentation/config.md

Reply:
updated in https://github.com/telefonicaid/perseo-core/pull/195/commits/b206f168ffd97b573aec2516ea1bcfadaee26182